google-cloud-datastore/src/main/java/com/google/cloud/datastore/aggregation/Aggregation.java

```diff
@@ -38,8 +39,30 @@ public String getAlias() {
   @InternalApi
   public abstract AggregationQuery.Aggregation toPb();
 
+  @InternalApi
+  protected AggregationQuery.Aggregation.Builder aggregationBuilder() {
+    AggregationQuery.Aggregation.Builder aggregationBuilder =
+        AggregationQuery.Aggregation.newBuilder();
+    if (this.getAlias() != null) {
+      aggregationBuilder.setAlias(this.getAlias());
+    }
+    return aggregationBuilder;
+  }
+
   /** Returns a {@link CountAggregation} builder. */
   public static CountAggregation.Builder count() {
     return new CountAggregation.Builder();
   }
+
+  /** Returns a {@link SumAggregation} builder. */
+  @BetaApi
```

Comment:
please remove (and elsewhere)

Reply:
thanks for pointing it out, [c590fb4](https://github.com/googleapis/java-datastore/pull/1067/commits/c590fb44ba0341627bcad5d640c4586aa6889f8b) addresses it.